gulpfile.js

```diff
@@ -96,43 +93,21 @@ const build = {
       .pipe(csso())
       .pipe(dest(path.build + '/css'))
   },
-  jsDependencies: function() {
-    return src(jsDependencies, {
-        sourcemaps: true
-      })
-      .pipe(concat(filename.jsDependencies))
-      .pipe(dest(path.build + '/js', {
-        sourcemaps: true
-      }))
-  },
-  jsFiles: function() {
-    return src(jsFiles, {
-        sourcemaps: true
-      })
-      .pipe(concat(filename.jsFiles))
-      .pipe(uglify())
-      .pipe(dest(path.build + '/js', {
-        sourcemaps: true
-      }))
-  },
   js: function() {
-    return src([
-        path.build + '/js/' + filename.jsDependencies,
-        path.build + '/js/' + filename.jsFiles
-      ], {
-        sourcemaps: true
-      })
+    const noVendors = filter(jsFiles, {
+      restore: true
+    });
+
+    return src(jsDependencies.concat(jsFiles), {
+      sourcemaps: true
+    })
+      .pipe(noVendors)
+      .pipe(uglify())
```

Comment:
Just want to confirm something here. Can you confirm the `uglify` step is only uglifying (is that a word? 😆)  nightTab js files and not `html5sortable.min.js` and `invert.min.js`? (These dependencies are already uglified code.)

I've encountered bugs in build code (in other projects) caused by running uglify on already uglified code. So would like to avoid that.

Reply:
Eh, merging anyway.

Reply:
> (is that a word? 😆)

https://en.wiktionary.org/wiki/uglifying
Good enough I would say xD

> Can you confirm the uglify step is only uglifying (is that a word? 😆) nightTab js

I based it on https://www.npmjs.com/package/gulp-filter#filter-only which suggests that this is the behavior.

Besides I manually compared the fist line of `build/web/js/` with `/Usersnode_modules/html5sortable/dist/html5sortable.min.js` and, well, they are the same.

So, I confirm!

Reply:
> I've encountered bugs in build code (in other projects) caused by running uglify on already uglified code. So would like to avoid that.

I was wondering why you separated them. Interesting!
Also, a solution would have been to include the non minified version of html5sortable & invert and letting nightTab build do the magic.

Reply:
> I was wondering why you separated them. Interesting!
Also, a solution would have been to include the non minified version of html5sortable & invert and letting nightTab build do the magic.

Indeed, I had this thought. I'll do some tests and if it becomes a problem we can switch out to the non `.min.js` dependency files. 👍